### DISCUSSION TITLE
Classes inherited from Setter are no longer available.

### DISCUSSION THREAD

**Discussion by Spaceyodo** [AvaloniaUI/Avalonia#12126] [Q&A]
After upgrading to version 11.0.0, classes inherited from Setter are no longer available because SetterBase.Instance has become an internal method. Are there any other solutions in version 11.0.0 that achieve the effect of inheriting Setter?
Here is the EventSetter I used earlier [https://github.com/codingseb/Avalonia.EventSetter](EventSetter )

**Comment by timunie** [comment]
Avalonia.Behaviors is better suited for this imo.

**Reply by Spaceyodo** [reply]
Thank you.
